### ISSUE TITLE
Failing to retrieve tagged_users

### ISSUE THREAD

**Issue by tfcbertaglia** [instaloader/instaloader#532]
**Describe the bug**

**UPDATE**: I just noticed that some other tags are also missing, like ``is_ad``, ``caption_is_edited``, ``accessibility_caption`` etc.

When downloading posts with the flags --no-pictures and --no-videos, it is not possible to retrieve tagged_users (e.g. via ``load_structure_from_file().tagged_users``). 

If the posts are downloaded *with* pictures, ``tagged_users`` can be retrieved for roughly 10% of the total; and if downloaded with comments, almost 100%. 

**To Reproduce**
Try to access the ``tagged_users``attribute from a downloaded post with ``instaloader.load_structure_from_file``.

**Error messages and tracebacks**
```KeyError                                  Traceback (most recent call last)
/usr/local/lib/python3.7/site-packages/instaloader-4.2.10-py3.7.egg/instaloader/structures.py in _field(self, *keys)
    158             for key in keys:
--> 159                 d = d[key]
    160             return d

KeyError: 'edge_media_to_tagged_user'

During handling of the above exception, another exception occurred:

AttributeError                            Traceback (most recent call last)
<ipython-input-221-aa0fe1c257f1> in <module>
----> 1 t.tagged_users

/usr/local/lib/python3.7/site-packages/instaloader-4.2.10-py3.7.egg/instaloader/structures.py in tagged_users(self)
    277         try:
    278             return [edge['node']['user']['username'].lower() for edge in self._field('edge_media_to_tagged_user',
--> 279                                                                                      'edges')]
    280         except KeyError:
    281             return []

/usr/local/lib/python3.7/site-packages/instaloader-4.2.10-py3.7.egg/instaloader/structures.py in _field(self, *keys)
    160             return d
    161         except KeyError:
--> 162             d = self._full_metadata
    163             for key in keys:
    164                 d = d[key]

/usr/local/lib/python3.7/site-packages/instaloader-4.2.10-py3.7.egg/instaloader/structures.py in _full_metadata(self)
    143     @property
    144     def _full_metadata(self) -> Dict[str, Any]:
--> 145         self._obtain_metadata()
    146         assert self._full_metadata_dict is not None
    147         return self._full_metadata_dict

/usr/local/lib/python3.7/site-packages/instaloader-4.2.10-py3.7.egg/instaloader/structures.py in _obtain_metadata(self)
    134     def _obtain_metadata(self):
    135         if not self._full_metadata_dict:
--> 136             pic_json = self._context.get_json("p/{0}/".format(self.shortcode), params={})
    137             self._full_metadata_dict = pic_json['entry_data']['PostPage'][0]['graphql']['shortcode_media']
    138             self._rhx_gis_str = pic_json.get('rhx_gis')

AttributeError: 'Instaloader' object has no attribute 'get_json'
```
**Instaloader version**
4.2.10

**Comment by aandergr** [comment]
Could you show us the code that produces this error message, especially the call to instaloader.load_structure_from_file()?

**Comment by aandergr** [comment]
Closing this, because I cannot reproduce the error and the report lacks the invocation of `instaloader.load_structure_from_file()` and the error traceback looks like it's wrongly invocated (e.g. an instance of Instaloader passed for the context argument instead of InstaloaderContext).

> **UPDATE**: I just noticed that some other tags are also missing, like ``is_ad``, ``caption_is_edited``, ``accessibility_caption`` etc.

These are no Post properties. You can find a list of Post properties here: https://instaloader.github.io/as-module.html#instaloader.Post
